defaults/defaults.env

```diff
@@ -40,6 +40,12 @@ export SHARD_COMMITTEE_PERIOD="${SHARD_COMMITTEE_PERIOD:-256}"
 export SAMPLES_PER_SLOT="${SAMPLES_PER_SLOT:-8}"
 export CUSTODY_REQUIREMENT="${CUSTODY_REQUIREMENT:-4}"
 export DATA_COLUMN_SIDECAR_SUBNET_COUNT="${DATA_COLUMN_SIDECAR_SUBNET_COUNT:-128}"
-export MAX_BLOBS_PER_BLOCK="${MAX_BLOBS_PER_BLOCK:-6}"
+export MAX_BLOBS_PER_BLOCK_ELECTRA="${MAX_BLOBS_PER_BLOCK_ELECTRA:-9}"
+export TARGET_BLOBS_PER_BLOCK_ELECTRA="${TARGET_BLOBS_PER_BLOCK_ELECTRA:-6}"
+export MAX_BLOBS_PER_BLOCK_EIP7594="${MAX_BLOBS_PER_BLOCK_EIP7594:-12}"
+export TARGET_BLOBS_PER_BLOCK_EIP7594="${TARGET_BLOBS_PER_BLOCK_EIP7594:-9}"
 export EL_PREMINE_ADDRS="${EL_PREMINE_ADDRS:-{}}"
+export MAX_REQUEST_BLOCKS_DENEB="${MAX_REQUEST_BLOCKS_DENEB:-128}"
+export MAX_REQUEST_BLOB_SIDECARS_ELECTRA=$(($MAX_REQUEST_BLOCKS_DENEB * $MAX_BLOBS_PER_BLOCK_ELECTRA))
+export MAX_REQUEST_BLOB_SIDECARS_EIP7594=$(($MAX_REQUEST_BLOCKS_DENEB * $MAX_BLOBS_PER_BLOCK_EIP7594))
```

Comment:
nice idea,  but we can't do this in here.
the defaults are loaded before any user defined value gets applied:
https://github.com/ethpandaops/ethereum-genesis-generator/blob/master/entrypoint.sh#L4

so the expressions will be evaluated with default values only.

this needs to happen somewhere in the entrypoint script, after all values are available :)